### PR TITLE
Added command to kill running process

### LINKED ISSUE #12: How do you stop a script?
I have `npm.runInTerminal` set to `false` so the output goes to `OUTPUT` pane inside Visual Studio Code editor. I'm running a watching + server script with npm script extension. How do I stop the server now?

### PR DESCRIPTION
**Pull request by Gigitsu**
Hello, with this pull request I want to resolve #12 .
Now you have a command to kill a running process with its own shortcut:
 * `cmd+r shift+x` for mac
 * `ctrl+r shift+x` for other.

I've added only one package in package.json ( `tree-kill` ) but since I've installed the package with `npm install --save tree-kill`,  npm has reformatted the whole file so there are a lot of different rows.

### CODE REVIEW

**Comment by egamma** [comment]
Thanks for the PR, that addresses a commonly requested feature.

- you only register process to kill when the script is not run in a terminal, that is, the setting `npm.runInTerminal` is set to false. This makes sense since the user can kill the process in the terminal. However, in this case the Command `terminate-script` is a no-op. However, when the setting is set to true, then you show the users that there are no processes to kill. I suggest to at least show the user a message like 'Killing is only supported when the setting 'runInTerminal' is 'false'.

File: src/main.ts [hunk 1]
```diff
@@ -5,12 +5,25 @@ import * as fs from 'fs';
 import { window, commands, workspace, OutputChannel, ExtensionContext, ViewColumn, QuickPickItem } from 'vscode';
 import { runInTerminal } from 'run-in-terminal';
 
+const kill = require('tree-kill');
```
Comment (egamma) [inline src/main.ts]: Let's use an import and make the kill module typed. 
import { kill } from 'kill-tree';

Add a kill-tree.d.ts to the typings folder:
declare module "kill-tree" {
	export function kill(pid: number, signal?: string, callback?: ()=>any);
}

**Comment by egamma** [comment]
Also please update the README.md

**Comment by Gigitsu** [comment]
Ok, thank you for the merge.
I will make another PR with your suggestions as soon as I can.

**Comment by Gigitsu** [comment]
I've updated my pull request. Thank you for the suggestion.

**Comment by egamma** [comment]
Thanks for updating. I've merged the PR and will publish a new version of the extension soon.

**Comment by egamma** [comment]
Published a new version of the extension.

**Pull request merged**